borg/archiver.py

```diff
@@ -731,6 +741,10 @@ def format_cmdline(cmdline):
         stats = archive.calc_stats(cache)
         print('Name:', archive.name)
         print('Fingerprint: %s' % hexlify(archive.id).decode('ascii'))
+
+        if archive.metadata.get(b'comment') is not None:
+            print('Comment:', archive.metadata[b'comment'])
```

Comment:
we could also use a default of '' and just always print it.
for old archives without comment feature or new archives with no comment, it would just print "Comment: " then.